README.md

```diff
@@ -54,7 +62,7 @@ npm run download
 
 ## or
 
-npm run download -- --admin-only # to only download hierarchy data, and not venues (venues require around 100GB of disk space)
+npm run download -- --admin-only # to only download hierarchy data, and not venues and postalcodes (venues require around 100GB of disk space)
```

Comment:
the note about disk space can probably be removed here, since the next section includes a warning about disk space